src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -221,12 +225,17 @@ export class IeObjectsController {
 	public async getSimilar(
 		@Headers('referer') referer: string,
 		@Param('id') id: string,
-		@SessionUser() user: SessionUserEntity
+		@SessionUser() user: SessionUserEntity,
+		@Query() similarQueryDto: SimilarQueryDto
 	): Promise<IPagination<Partial<IeObject>>> {
 		const visitorSpaceAccessInfo =
 			await this.ieObjectsService.getVisitorSpaceAccessInfoFromUser(user);
 
-		const similarIeObjects = await this.ieObjectsService.getSimilar(id, '_all', referer);
+		const similarIeObjects = await this.ieObjectsService.getSimilar(
+			id,
+			similarQueryDto.maintainerId ? similarQueryDto.maintainerId.toLowerCase() : '_all',
```

Comment:
this should be the same behavior:
```suggestion
			similarQueryDto.maintainerId?.toLowerCase() ?? '_all',
```